web/server/vue-cli/src/views/NewFeatures.vue

```diff
@@ -2,6 +2,170 @@
+                with the information about the recently introduced or resolved
+                reports or about the distribution of the reports in the
+                product.
+              </li>
+              <li>
+                The <strong>Run History</strong> list was moved from a separate
+                tab to an <strong>expandable list</strong> under each run at
+                the run list. This makes easier to find the relevant run
+                history entries for each run.
+              </li>
+              <li>
+                New <strong>Report Info Button</strong> to show more
+                information about a report at the report details page
+                (run name, detection/fix date ...).
+              </li>
+              <li>
+                Source components can be used to create and save file path
+                filters with a name to show results only from those parts
+                of the analyzed project. With the newly introduced
+                <strong>Other component</strong> every report which does not
+                belong to any other component can be filtered.
+              </li>
+            </ul>
+          </new-feature-item>
+
+          <new-feature-item>
+            <template v-slot:title>
+              Command line interface (CLI)
+            </template>
+
+            <ul>
+              <li>
+                <strong>New exit status</strong> numbers for the CodeChecker
+                analyze and check commands for better <i>CI integration</i>:
+                <ul>
+                  <li>
+                    0 - Successful analysis and no new reports
+                  </li>
+                  <li>
+                    1 - CodeChecker error
+                  </li>
+                  <li>
+                    2 - At least one report emitted by an analyzer and there
+                    is no analyzer failure
+                  </li>
+                  <li>
+                    3 - Analysis of at least one translation unit failed
+                  </li>
+                </ul>
+              </li>
+              <li>
+                <strong>Gerrit output</strong> format is available
+                <strong>for the parse subcommand</strong>.
+                This output format was only available for the CodeChecker
+                cmd diff command in the previous releases. With this change
+                the parse command can be used for the gerrit integration
+                too.<br>
+                <code>CodeChecker parse analyzer_reports -e gerrit</code>
+              </li>
+            </ul>
+          </new-feature-item>
+
+          <new-feature-item>
+            <template v-slot:title>
+              Report storage support for new source code analyzers
+            </template>
+
+            <ul>
+              <li>
+                Report conversion and storage support is available for multiple
```

Comment:
I think it would be worth to add links to the tools and at least one link to the report converter documentation so the users can easily find it.